tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LRATypeTckResource.java

```diff
@@ -34,6 +34,7 @@
 @ApplicationScoped
 @Path(LRATypeTckResource.TCK_LRA_TYPE_RESOURCE_PATH)
 public class LRATypeTckResource {
+
```

Comment:
I personally prefer the spacing style that is changed in this file and presumably it's not broken the checkstyle? But it seems like something that should be considered for future checkstyle rules to help keep patches clearer. In this case the file is connected to being changed by issue214 but it is just formatting from what I can see?

Reply:
checkstyle passed. There shouldn't be any problems with formatting. But the formatting rules is something that we can revisit.